source/gems/dry-logic/predicates.html.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Predicates
+layout: gem-single
+---
+
+Dry-logic comes with a lot predicates to compose multiple rules:
+
+``` ruby
+require 'dry/logic'
+require 'dry/logic/predicates'
+
+include Dry::Logic
+```
+
+Now you can access all built-in predicates:
+
+``` ruby
+Predicates[:key?]
+=> #<Method: Module(Dry::Logic::Predicates::Methods)#key?>
+```
+
+In the end predicates return a called method:
+
+```ruby
+Predicates[:key?].(:name, {name: 'John'})
+=> true
+
+# You can even create your predicates at runtime
+Predicates[:predicate].(:coll_eh) { puts 'Woow 0_o' }
+=> :coll_eh
+Predicates[:coll_eh].()
+Woow 0_o
```

Comment:
Let's skip this part for now. Registering new predicates shouldn't be a common thing to do.